src/resources/elements/dropdown/dropdown.ts

```diff
@@ -19,7 +19,31 @@ export class Dropdown {
   private titleText: string;
 
   async attached(): Promise<void> {
-    this.title.addEventListener("click", (e) => this.toggleMenuDisplay(e));
+    document.addEventListener("click", (e: MouseEvent) => {
+      // Close the menu if the user clicks outside the dropdown
+      let isInDropdown = false;
+      const clickedElement = e.target as HTMLElement;
+      if (this.dropdown) {
+        this.dropdown.childNodes.forEach(node => {
+          if (isInDropdown) return;
+          isInDropdown = node.nodeType === 1 && node.textContent.includes(clickedElement.textContent);
```

Comment:
note that Aurelia has a @children (or something like that) decorator for populating a variable with the child elements of the view.

Reply:
1. Simplified the check for the dropdown. I use `.closest()` method of the Element interface. Seems cleaner to me, what do you think?
2. I've also commented out the `max-height` property, in order to remove the scroll bar. I've left the rest of the styling meanwhile, in case we'll might want to use it in the future.